Eastern-Research-Group/expert-query
# Feature/bean count endpoint

## Related Issues:
* [EQ-266](https://jira.epa.gov/browse/EQ-266)

## Main Changes:
* Added new `<profile>/countPerOrgCycle` endpoints which return the number of records for each distinct combination of cycle id, organization id and reporting cycle.
  * Note: For Actions and TMDL profiles this just returns the number of records for each organization id. 
* Added code to automatically parse numeric data types coming from the database, to ensure the service returns the values in the correct datatype. Before we were manually calling `parseInt` for the `/count` endpoints and now we no longer need to do that.

## Steps To Test:
1. Run the following in the browser console for each profile: `fetch('http://localhost:9090/api/attains/actions/countPerOrgCycle', { method: 'GET', headers: {'X-Api-Key': 'ULMokKTtpbb6N5hesW8UJ2qRLxEbsM3NFcehqQ4I'}}).then(res=>res.json()).then(console.log);`
    * Verify the results look accurate
    * Note: For actions and tmdl you will only see the `organizationid` and `count` keys. All other profiles will return `organizationid`, `cycleid`, `reportingcycle` and `count`.
2. Verify the api documentation for these new endpoints looks correct and works.
3. Verify the `<profile>/count` endpoints are still returning an integer
4. Verify that you can still download results
